.gitignore

```diff
@@ -3,7 +3,7 @@ target/
 !.mvn/wrapper/maven-wrapper.jar
 !**/src/main/**/target/
 !**/src/test/**/target/
-.env
+.env.temlate
```

Comment:
Why do you ignore the template file?

Reply:
fix comment
you should ignore env file, not env.template

Reply:
remove `.env.temlate`